### DISCUSSION TITLE
Vue.js support?

### DISCUSSION THREAD

**Discussion by entity** [recharts/recharts#2993] [Ideas]
Hey there,

Is there any planned support for Vue.js? If not, do any unofficial ports exist?

**Comment by ckifer** [comment]
Hey @heychazza,

At this point there isn’t much planned at all due to the maintainers not being active. In the code Recharts is relatively React specific. If eventually there is enough interest and the repo moves forward then it’d be awesome to get a port or support underway.

**Comment by ckifer** [comment]
Coming back to this repo after a while. Recharts is incredibly React specific, a port to Vue might as well be another charting library written using d3. That being said there won't be a Vue port for Recharts unless someone starts a separate lib.
